src/data/linux-quiz.ts

```diff
@@ -717,6 +717,39 @@ const linuxQuiz = [
       "Absolute mode uses numbers to represent permissions and mathematical operators to modify them.",
     Link: "https://www.freecodecamp.org/news/linux-chmod-chown-change-file-permissions/",
   },
+  {
+    Question:
+      "In Linux, which command is used to display network status and generate protocol statistics?",
+    Answer: "netstat",
+    Distractor1: "ipconfig",
+    Distractor2: "ifconfig",
+    Distractor3: "netlist",
+    Explanation:
+      "Netstat command displays various network related information such as network connections, routing tables, interface statistics, masquerade connections, multicast memberships etc.,",
```

Comment:
It looks like this explanation is a direct quote from this geeks for geeks article
https://www.geeksforgeeks.org/netstat-command-linux/

Please rewrite this explanation in your own words.